stdlib/public/core/StringStorage.swift

```diff
@@ -182,7 +182,10 @@ extension _SwiftStringStorage {
   @inlinable
   @nonobjc
   var unusedBuffer: UnsafeMutableBufferPointer<CodeUnit> {
-    return UnsafeMutableBufferPointer(start: end, count: capacity - count)
+    @inline(__always)
```

Comment:
Why is this necessary? What model can we use to determine when we should flag something as `@inline(__always)`?

Reply:
In this case I just wanted to ensure that this is always inlined - also with future compiler versions.
There is no general model.

Reply:
https://bugs.swift.org/browse/SR-7635 😀